server/src/streaming/systems/info.rs

```diff
@@ -166,3 +166,15 @@ impl Display for SemanticVersion {
         )
     }
 }
+
+mod tests{
+
+    use super::*;
+
+    #[test]
+    fn test_1(){
```

Comment:
Please use the meaningful test names, whether you prefer following e.g. BDD naming convention (Given-When-Then) or anything else, it's up to you :) For example, `should_load_the_expected_version_from_package_definition`, `given_environment_variable_with_package_version_should_load_the_expected_version` or something similar that represents the test use case.